eups.tex

```diff
@@ -2,19 +2,19 @@ \subsection{EUPS}\label{sec:eups}
 
 Managing the dependencies of many separate packages all undergoing rapid change including changing APIs, is a hard problem.
 At LSST we use the Extended Unix Product System (EUPS)\cite{EUPS}.
-EUPS was initially developed for SDSS \{citation needed\} as UPS, and has evolved over time migrating from Perl to Python.
-The EUPS philosophy is to organize the hierarchy of components that are required to make a release of a complex software product.
-It does this by understanding the dependencies between packages and knowing which versions of those packages were built together.
-As products are built they are installed into a directory tree where each built product is isolated from other products and all built versions of a product are available.
-A database, currently consisting of files using a bespoke file format, knows which version of a product should be used with specific versions of other products such that a coherent environment can be presented to the user.
-When a user requests that a particular product be ``set up'', EUPS sets environment variables in the current shell for every product that is required.
-Each product can specify which environment variables are relevant (for example \texttt{LD\_LIBRARY\_PATH}, \texttt{PATH}, \texttt{PYTHONPATH}) in a ``table'' file which is also used to declare optional and required dependencies.
-For each product that is being set up, EUPS adjusts the environment and prepends or appends to any previous value as requested.
+EUPS was initially developed at FNAL and adopted by SDSS \{citation needed\} as UPS, and has evolved over time migrating from C to Perl to Python.
```

Comment:
I'll drop the citation since we can't really find anything public that mentions this.